tools/topology/sof-jsl-rt5682.m4

```diff
@@ -62,12 +62,24 @@ PIPELINE_PCM_ADD(sof/pipe-volume-playback.m4,
 	1000, 0, 0,
 	48000, 48000, 48000)
 
+ifdef(`INCLUDE_IIR_EQ',
+`
 # Low Latency playback pipeline 2 on PCM 1 using max 2 channels of s24le.
```

Comment:
this comment is wrong, mind updating it?

Reply:
all done!